index.html

```diff
@@ -131,7 +133,7 @@ <h2>Tonic</h2>
                             </div>
                         </div>
                         <div class="card-button">
-                            <button class="button-work-project">SEE PROJECT</button>
+                            <button class="button-work-project" id="projectOne">SEE PROJECT</button>
```

Comment:
- [ ] Great job making changes requested by the previous reviewer 💯, but as @Reem-lab says [here](https://github.com/gjuliao/myproject/pull/13#discussion_r1024593837),  you should create the projects section dynamically using the information provided by the array from `projects.json`.

![image](https://user-images.githubusercontent.com/91301423/202181640-409e92bc-1bbe-4e36-a75f-bf33a411648e.png)


To create it dynamically you could remove the HTML code used for the projects section and create it using JavaScript code, and after that, you could implement the popup window, here you have an [example](https://github.com/olawale-o/olawale-o.me/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95). 🦘